ledger/participant-state/kvutils/tools/src/main/scala/com/daml/ledger/participant/state/kvutils/tools/driver/IntegrityCheckV2.scala

```diff
@@ -32,12 +31,12 @@ object IntegrityCheckV2 {
     val executionContext: ExecutionContextExecutorService =
```

Comment:
Having `IntegrityCheckV2` work with `v3.ProtobufBasedLedgerDataImporter` is confusing.

Reply:
I totally agree. I'd rather have one integrity checker application that worked regardless of version.